aiida_quantumespresso/calculations/pp.py

```diff
@@ -153,17 +158,23 @@ def prepare_for_submission(self, folder):  # pylint: disable=too-many-branches,t
                 parameters[namelist] = {}
             parameters[namelist][key] = value
 
-        # Restrict the plot output to the file types that we want to be able to parse
-        dimension_to_output_format = {
-            0: 0,  # Spherical integration -> Gnuplot, 1D
-            1: 0,  # 1D -> Gnuplot, 1D
-            2: 7,  # 2D -> Gnuplot, 2D
-            3: 6,  # 3D -> Gaussian cube
-            4: 0,  # Polar on a sphere -> # Gnuplot, 1D
-        }
-        parameters['PLOT']['output_format'] = dimension_to_output_format[parameters['PLOT']['iflag']]
-
-        namelists_toprint = self._default_namelists
+        if skip_plot:
+            parameters.pop('PLOT')
+            namelists_toprint = list(self._default_namelists)
+            if 'PLOT' in namelists_toprint:
+                namelists_toprint.remove('PLOT')
```

Comment:
If I'm just doing step1, then I just remove the `PLOT` from the parameters (it is there because there were some defaults loaded in after checking the user didn't set anything themselves) and from the list of `namelists_toprint`.

I need to remove it from `namelists_toprint` because the program iterates there to know what namelists to look inside the parameters and print them in the input file (it pops them out of the parameters).

I need to remove it from `parameters` because it will be checked after setting the input for namelists that have not been poped out and will except.